osbs/build/build_request.py

```diff
@@ -26,7 +25,7 @@
 from osbs.build.spec import BuildSpec
 from osbs.constants import SECRETS_PATH, DEFAULT_OUTER_TEMPLATE, DEFAULT_INNER_TEMPLATE, DEFAULT_CUSTOMIZE_CONF
 from osbs.exceptions import OsbsException, OsbsValidationException
-from osbs.utils import looks_like_git_hash, git_repo_humanish_part_from_uri
+from osbs.utils import looks_like_git_hash, git_repo_humanish_part_from_uri, sanitize_version
```

Comment:
I'm told up to 99 chars is allowed in this project, and looks_like_git_hash was already there, not something I'm adding. It could be removed, but that's really not part of this change.